### DISCUSSION TITLE
Trouble in finding the slack channel for 2024 Outreachy applicants

### DISCUSSION THREAD

**Discussion by Ashita17** [ModECI/MDF#502] [General]
Hello mentors. I am Ashita Srivastava and I am an Outreachy applicant for the May 2024 to August 2024 Internships. Is there any specific channel for 2024 interns?

**Comment by Lily-kwamboka** [comment]
Hello mentors as well as @Ashita17. I am Lilian Kwamboka and I am interested to make a contribution in this cohort. I would like to get to know the way forward.

**Reply by chickenleaf** [reply]
hello @Ashita17 as of now i think there is no stack channel made for this.

**Reply by Ashita17** [reply]
okay

**Comment by pgleeson** [comment]
Hi all! See https://github.com/ModECI/MDF/discussions/509 for the main place to discuss the project.
